src/components/ProjectCard/ProjectCard.module.css

```diff
@@ -0,0 +1,109 @@
+body {
+  background-color: black;
+}
+
+h5 {
+  color: rgb(51, 255, 0);
+}
+
+.workcard {
+  width: 19rem;
+  height: 370px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  background-color: rgb(111, 175, 231);
+  border: 4px solid;
+  border-radius: 23px;
+  border-color: greenyellow;
+  vertical-align: middle;
+  transform: translateZ(0);
+  box-shadow: 0 0 1px rgba(0, 0, 0, 0);
+  backface-visibility: hidden;
+  -moz-osx-font-smoothing: grayscale;
+  transition-duration: 0.3s;
+  transition-property: transform;
+}
+
+.card-title {
+  text-align: center;
+  font-size: 19px;
+  font-weight: bold;
+  text-shadow: 4px 3px 10px black;
+}
+
+.card-text {
+  font-weight: bold;
+  text-align: center;
+}
+
+.card:hover,
+.card:focus,
+.card:active {
+  /* box shadow */
+  -webkit-box-shadow: 1px 5px 24px -1px rgb(120, 251, 120);
+  -moz-box-shadow: 1px 5px 24px -1px rgb(120, 251, 120);
+  box-shadow: 1px 1px 20px -1px rgb(120, 251, 120);
+  transform: scale(1.1);
+}
+
+.imgWork {
+  width: 310px;
+  height: 165px;
+}
+
+.workButton {
+  width: 15rem;
+  display: block;
+  background-color: greenyellow;
+  font-family: "Poppins";
+  border-radius: 8px;
+  padding: 2px 8px;
+  text-decoration: none;
+}
+
+.card-body {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px;
+  flex: 1;
+}
+
+.hcj {
+  background-color: crimson;
+}
+
+@media (max-width: 768px) {
```

Comment:
Please define all the break points at one place and please refer those everytime

Reply:
Please try using globally accepted break-points like defined by MUI or Bootstrap

Reply:
ok will do this change in a separate pr since this will need to be changed throughout the project